configs/edison/config.20161006bugfix.alpha8.A_WCYCL1850S.ne30_oEC_ICG.edison

```diff
@@ -131,6 +131,7 @@ polarPlot = False
 [streamfunctionMOC]
 ## options related to plotting the streamfunction of the meridional overturning 
 ## circulation (MOC)
+maxChunkSize = 1000
```

Comment:
For 10-year climatologies, 10,000 was fine.  For longer, I couldn't find anything that took care of the memory problem.  So I am not convinced that maxChunkSize has a good choice on Edison.  (Sounds similar for other machines.)